.circleci/config.yml

```
@@ -326,7 +326,21 @@ jobs:
           channel: '#sh_releases'
           color: '#2EB67D'
           title: $(cat .version)
-          message: $(cat changelogs/changelog.txt)
+          message: |
+            Version $(cat .version) of the Codacy chart has been released!
+            Release on GitHub: https://github.com/codacy/chart/releases/tag/$(cat .version)
+            Release notes: https://github.com/codacy/chart/releases/download/$(cat .version)/changelog.md
+
+  publish_changelog_to_github:
+    <<: *default_doks_image
+    steps:
+      - <<: *attach_workspace
+      - run:
+          name: "Install requirements"
+          command: curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1 && mv bin/hub /usr/local/bin/hub
+      - run:
+          name: Update github release
+          command: hub release edit -a changelogs/changelog.md -m "$(date "+%d-%m-%Y")" -m "Self-hosted helm chart for Codacy." $(cat .version)
```

Use [ISO 8601 date format](https://en.wikipedia.org/wiki/ISO_8601#Dates) instead:

```suggestion
          command: hub release edit -a changelogs/changelog.md -m "$(date "+%Y-%m-%d")" -m "Self-hosted helm chart for Codacy." $(cat .version)
```